crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -207,7 +207,7 @@ fn is_source(
     asset: &Denom,
     is_refund: bool,
```

Comment:
This isn't strictly speaking part of this PR, but this extra argument is very confusing/surprising and only really used in one spot (the function is called in 2 places, but in the argument is always `true`).

I'd remove the argument in favor of just handling this case there.

Reply:
the argument isn't always true, it's false when called from recv_packet_execute -> execute_ics20_transfer

Reply:
It's used in two places:
1. Here it's hardcoded with `true`: https://github.com/astriaorg/astria/blob/561572ce1600092b1af0a036527616fb98088641/crates/astria-sequencer/src/ibc/ics20_transfer.rs#L183
2. Here it's actually a variable: https://github.com/astriaorg/astria/blob/561572ce1600092b1af0a036527616fb98088641/crates/astria-sequencer/src/ibc/ics20_transfer.rs#L345